packages/widget-v2/package.json

```diff
@@ -79,7 +80,7 @@
     "@tanstack/react-query": "^5.51.21",
     "chain-registry": "^1.63.43",
     "graz": "^0.1.25",
-    "jotai": "^2.9.1",
+    "jotai": "^2.10.1",
```

Comment:
updated jotai version

Reply:
interestingly, after updating the jotai version i was able to see the stack size exceeded in the devmode / http://localhost:5173